plugins/pulp_docker/plugins/distributors/publish_steps.py

```diff
@@ -14,6 +14,9 @@
 from pulp_docker.plugins.distributors import configuration, v1_publish_steps
 
 
+tags_names_redirect = {}
```

Comment:
Since within PublishTagsStep we are collecting the tags names for both schema versions, i needed to make this data available to  RedirectFileStep to it can use it later.
Both steps eventually point to the same parent class from which they are inherited, but i did not want to make a change in platform by adding a new param.